contribute/partest-guide.md

```diff
@@ -3,15 +3,35 @@ layout: page
 title: Running the Test Suite
 ---
 
-Partest is a custom parallel testing tool that we use to run the test suite for the Scala compiler and library. Go the scala project folder from your local checkout and run it via `ant` or standalone as follows.
+Partest is a custom parallel testing tool that we use to run the test suite for the Scala compiler and library. Go to the scala project folder from your local checkout and run it via `sbt`, `ant` or standalone as follows.
+
+## Using sbt
+
+The test suite can be run from the sbt console with:
+
+	> partest
+
+You can get a summary of the usage by running `partest --help`.
+
+If you would like to run particular tests pass the test paths as arguments
+
+        > partest test/files/pos/bounds.scala test/scaladoc/run/diagrams-base.scala
+
+To run only the Scaladoc tests use `--srcpath` with the location of the tests
+
+        > partest --srcpath scaladoc
```

Comment:
Definitely has a benefit but I'm happy to leave a fuller update to a separate PR. This is about plugging a gap related to 2.12 and sbt rather than documenting partest.